### ISSUE TITLE
ASUS Ryzen "Invalid Password"

### ISSUE THREAD

**Issue by brbsix** [ChubbyAnt/sedutil#15]
I'm having an issue with an ASUS notebook with an AMD Ryzen 5 3500U and a Samsung 860 EVO. It occurs with your fork and also upstream but I hoped that you may have some insight. I can go through the drive setup and testing as usual without any issue. No problems with `linuxpba`.

However upon reboot, the BIOS prompts me to enter the user password for the drive. I enter the password *debug* and it reports an invalid password. After a few more tries, it prompts for the admin password for the drive. Similarly invalid. After a few tries I'm locked out.

I also tried with a password of *123456* and other similarly trivial ones. No luck.

This password prompt occurs even before I can get into the BIOS menu. It occurs even if the SSD is at the bottom of the boot priority, below a RESCUE64 thumb drive and the original Windows installation.

I can unlock the drive by getting locked out then booting straight into the rescue drive. Then unplug the SSD then plug it back in. From there I can unlock it.

At this point I'm not sure what else I can do. It seems like an issue with the BIOS, possibly an insurmountable one. Any ideas?

![BIOS SSD Login Screen](https://i.ibb.co/NyyQVNy/login.jpg)

![BIOS SSD Locked Out Screen](https://i.ibb.co/gRnvcYm/locked.jpg)

**Comment by ChubbyAnt** [comment]
What model is this machine, what chip and chipset? 

Were you able to resolve this issue?

**Comment by brbsix** [comment]
No, I wasn't able to resolve the issue. One other thing to note, I have another Samsung 860 EVO drive that was set up, locked, and working on another machine. I put it in the notebook and attempted to boot from it but was immediately prompted to unlock it via the same blue BIOS prompt. No luck.

Its an ASUS VivoBook 15 F512DA. `dmidecode` reports the model name as VivoBook_ASUSLaptop X512DA_F512DA. It has an AMD Ryzen 5 3500U CPU. I'm not sure what the chipset is.

**Comment by neo125874** [comment]
@brbsix Hi, did you resolve this issue? or any event to trigger this popup?

cause we have encountered this issue after locking one nvme drive on intel nuc platform only once, 
but couldn't reproduce it by locking the drive again.
